### DISCUSSION TITLE
Is the angular diameter distance turnaround real?

### DISCUSSION THREAD

**Discussion by mikehelland** [a-cosmology-group/acg#165] [Cosmology Models]
Made a video about the surprising feature of LCDM that is the angular diameter distance turnaround:

https://www.youtube.com/watch?v=tvRsXYHW2RU

It seems our attempts to verify this with evidence fail pretty badly. 

Are there more current, larger datasets available that you know of?

![](https://raw.githubusercontent.com/mikehelland/hubbles-law/master/img/addturnaround.gif)

**Comment by RedshiftDrift** [comment]
No turnaround is observed, ΛCDM is a farce.  See Fig. 5 in N. Lovyagin, A. Raikov, V. Yershov, and Y. Lovyagin, “Cosmological Model Tests with JWST,” Galaxies, vol. 10, no. 6, Art. no. 6, Dec. 2022, doi: [10.3390/galaxies10060108](https://doi.org/10.3390/galaxies10060108).

(By the way, some links in the comment section of your YouTube video are incomplete.)

![angular-diameter](https://github.com/a-cosmology-group/acg/assets/16748402/63d8867a-0e43-42db-983d-9edee8fd97c9)
